appengine/standard_python37/django/app.yaml

```diff
@@ -0,0 +1,16 @@
+# [START django_app]
+runtime: python37
+entrypoint: gunicorn -b :$PORT mysite.wsgi
```

Comment:
Is there any way we can do this without specifying a manual entrypoint, instead using the default entrypoint?

If there is not (I know Django is more challenging in this respect) then please change this entrypoint line to add multiple workers and threads. We should make every effort to use the default without an entrypoint though, even if it is awkward.